sop/modules/OSD/partials/pod_debug.adoc

```diff
@@ -5,7 +5,7 @@
 .pod status: pending -> Check for resource issues, pending pvcs, node assignment, kubelet problems.
 ----
 oc project openshift-storage
-oc get pod | grep {ceph-component}
+oc get pod | grep rook-ceph-mon
```

Comment:
Is it always the mon pod in this section? or mon pod as example?

Reply:
as per this sop the rook-ceph-mon is the one updating, i removed {ceph-component} as it was equivalent to the value "rook-ceph-mgr" and there were a lot of consumers for {ceph-component}